src/lib/optimizer/strategy/stored_table_column_alignment_rule.cpp

```diff
@@ -86,4 +96,32 @@ void StoredTableColumnAlignmentRule::apply_to(const std::shared_ptr<AbstractLQPN
   }
 }
 
+}  // namespace
+
+namespace opossum {
+
+/**
+ * The default implementation of this function optimizes a given LQP and all of its subquery LQPs individually.
+ * However, as we do not want to align StoredTableNodes per plan but across all plans, we override it accordingly.
+ */
+void StoredTableColumnAlignmentRule::apply_to_plan(const std::shared_ptr<LogicalPlanRootNode>& root_node) const {
+  // (1) Collect all plans
+  auto lqps = std::vector<std::shared_ptr<AbstractLQPNode>>{std::static_pointer_cast<AbstractLQPNode>(root_node)};
```

Comment:
I'd stick to doing one thing per line. Took me a second to check whether the root node was added.